lambdas/helpers.py

```diff
@@ -54,7 +55,7 @@ def generate_step_output_filename(
     """
     sequence_suffix = f"_{sequence}" if sequence else ""
     if step == "extract":
-        file_type = "xml"
+        file_type = "jsonl" if "gis" in source else "xml"
```

Comment:
To avoid the risk of the string `"gis"` being present in another source name -- current or future -- I'd propose that we make this an explicit list like you have other places, e.g.

```python
if source in ["gismit", "gisogm"]:
    # logic here to set .jsonl file extension
```

Reply:
Updated!